code/clue.js

```diff
@@ -40,20 +163,68 @@ const randomSelector = array => {
 // CREATE AN OBJECT THAT KEEPS THE MYSTERY.
 // With a killer, a weapon and a room.
 // The values will be set later.
+const mystery = {
+  suspects,
+  weapons,
+  rooms:[]
```

Comment:
Why do you initialise your mystery object with arrays of all suspects, weapons and empty array of rooms?
At the beginning it should be empty, and later on you should add to each property (so, suspect, weapon and rooms (please, notice that these are single, not multi)) value as single element from suspets, weapons and rooms array. So

1. Beginning :
```
const mystery = {
  suspect: null,
  weapon: null,
  rooms: null
}
```

2. Finish :
```
mystery = {
  suspect: //Mr green object,
  weapon: // knife object,
  rooms: // dining room object
}
```